CHANGELOG.md

```diff
@@ -61,6 +56,7 @@
 * [ENHANCEMENT] Distributor: Drop exemplars in distributor for tenants where exemplars are disabled. #2504
 * [ENHANCEMENT] Runtime Config: Allow operator to specify multiple comma-separated yaml files in `-runtime-config.file` that will be merged in left to right order. #2583
 * [ENHANCEMENT] Query sharding: shard binary operations only if it doesn't lead to non-shardable vector selectors in one of the operands. #2696
+* [ENHANCEMENT] Add packaging for both debian based deb file and redhat based rpm file using FPM. #1803
```

Comment:
```suggestion
* [ENHANCEMENT] Add packages for Debian and CentOS. #1803
```

Reply:
We're not supplying the packages, just added the ability to create them in that PR.

Reply:
I think the plan is to use that tooling to publish them, no?

Reply:
> I think the plan is to use that tooling to publish them, no?

That's my understanding too. We may want to update RELEASE.md to mention build of packages during release process.

Reply:
We can do that, I'll build those and add it to the release process!